stitch/cli.py

```diff
@@ -46,18 +48,39 @@ def enhance_args(to, no_standalone, no_self_contained, extra_args):
 )
 @click.pass_context
 @click.argument('input_file', type=click.File('rb'))
+@click.option('-y', '--yaml', type=click.File('rb'), default=None)
 @click.option('-o', '--output_file', type=str, default=None)
 @click.option('-t', '--to', default=None)
 @click.option('--no-standalone', is_flag=True, default=False,
               help='Produce a document fragment instead.')
 @click.option('--no-self-contained', is_flag=True, default=False,
               help='Use external files for resources like images.')
-def cli(ctx, input_file, output_file, to, no_standalone, no_self_contained):
+def cli(ctx, input_file, yaml, output_file, to, no_standalone, no_self_contained):
     if to is None:
         to = infer_format(output_file)
     input_text = input_file.read().decode('utf-8')
+    input_file_path = os.path.abspath(input_file.name)
+    yaml_text = None
+    if yaml is not None:
+        yaml_text = yaml.read().decode('utf-8')
     extra_args = enhance_args(to, no_standalone, no_self_contained, ctx.args)
-    convert(input_text, to, output_file=output_file, extra_args=extra_args)
+
+    # Atom compatibility patch: for some reason RTerm.exe does not close
```

Comment:
Could you link to the issue tracking this in node / rterm? This adds a lot of complexity I'd like to avoid by fixing it upstream if possible.